communities/aston-css/community-assessment.md

```diff
@@ -0,0 +1,159 @@
+
+- Are they students hoping to get a degree?
+
+Yes.
+
+- Are they trying to ramp-up to get a job?
+
+We do hold company sponsored tech talks (read: employment scouting events).
+
+- Are they passionate about developing, regardless of their major?
+
+Yes! In fact, we sometimes see members from outside of the CS degree being more excited about some of the stuff we're running than those doing the degree! I think this is partly because we seem to have many people doing CS at Aston who kind of drifted into it
+- they don't really have much passion and are almost just in it for the money. Maybe we should aim to increase interest in more subjects outside of the course.
+
+- What activities and goals can you set for your community based on their existing goals?
+
+Continue hosting weekly events! I was part of a push around February to get a regular schedule of events going, and I'd like to keep it up. Also make sure revision sessions run
+- these have been really useful for many members in the past, but we unfortunately didn't run them in January due to lack of people willing to host them.
+
+### Location
+
+- Where does your community currently gather?
+
+We have a Facebook group which gets a lot of activity from our regulars, and also an actual, physical society room (we always have, but this year we got a super-duper new one!) which again, gets many of the regulars. We also sometimes see non-members in there who have been brought in by a friend or for group project or something.
+
+- Is it an online-only group so far, or is there a place where people can meet?
+
+See above (we have a room).
+
+- Is there parking, and if so, is it free?
+
+Yes, and no. But this is not really relevant for most members.
+
+- Is there public transportation available to and from meet-ups?
+
+People are usually already in the University building so not applicable. We are a 10 minute (Scott pace) walk from the central train stations in Birmingham for those not on campus. Very accessible.
+
+- Is the location accessible to those both on and off campus?
+
+Yes, see above.
+
+- How much does it cost to use the space?
+
+Totally free!
+
+- How flexible is the space for different types of presentations and meetings?
+
+It has a whole load of chairs and tables, but they're all on legs and can be moved about very easily (and are regularly). We can expand the space if needed, and occupy the student software company office (there's a big double door that opens up to join the two). The room has a projector and speakers.
+
+- How many people can it fit?
+
+I've never tried to cram it full before, but maybe 30 people before it gets too much. And that's a bit of a squeeze. As a result, we quite often book out other rooms in the University for larger events. This works pretty well, especially now that we've got a contact in the department who's happy to book rooms for us any time, in an instant!
```

Comment:
It got very crowded with 40 to the point where people couldn't easily manoeuvre around the room.